dask_jobqueue/tests/test_htcondor.py

```diff
@@ -26,6 +27,8 @@ def test_job_script():
         disk="100MB",
         env_extra=['export LANG="en_US.utf8"', 'export LC_ALL="en_US.utf8"'],
         job_extra={"+Extra": "True"},
+        submit_command_extra=["-verbose"],
```

Comment:
I would actualy use one test with multiple arguments like ["-batchname","foo"] (I know its kinda rediculous, but it would test the join"